src/wallet/wallet2.h

```diff
@@ -1845,6 +1850,8 @@ namespace tools
     // If m_refresh_from_block_height is explicitly set to zero we need this to differentiate it from the case that
     // m_refresh_from_block_height was defaulted to zero.*/
     bool m_explicit_refresh_from_block_height;
+    uint64_t m_pool_info_query_time;
+    std::vector<std::tuple<cryptonote::transaction, crypto::hash, bool>> m_process_pool_txs;
```

Comment:
These fields cache RPC request information. Put these fields inside `NodeRPCProxy` and invalidate them inside of the `NodeRPCProxy::invalidate` method. Different nodes may have different clock times (or steady clocks in future PRs). When the daemon is switched out, so should this info.